devicetypes/smartthings/cree-bulb.src/cree-bulb.groovy

```diff
@@ -25,7 +25,7 @@ metadata {
         capability "Health Check"
         capability "Light"
 
-        fingerprint profileId: "C05E", inClusters: "0000,0003,0004,0005,0006,0008,1000", outClusters: "0000,0019"
+        fingerprint manufacturer: "CREE", model: "Connected A-19 60W Equivalent" // 0A C05E 0100 02 07 0000 1000 0004 0003 0005 0006 0008 02 0000 0019
```

Comment:
I needed to make this change to make sure Cree devices used this DTH instead of one of the other DTHs. However I still need to test to verify this works since Cree devices report somewhat weird manufacturer/model info.

Reply:
I successfully tested this with self published version of both `Cree Bulb` an `ZLL Dimmer Bulb`. I'll check it again once this goes to STG just to make sure there isn't some weird stuff going on with self published DTHs.